test/Engine/ProtoTestConsoleRunner/ProtoTestConsoleRunner.csproj

```diff
@@ -14,12 +14,9 @@
   <PropertyGroup>
     <StartupObject>ProtoTestConsoleRunner.Program</StartupObject>
   </PropertyGroup>
-  <PropertyGroup>
-    <SignAssembly>false</SignAssembly>
-  </PropertyGroup>
-  <PropertyGroup>
-    <AssemblyOriginatorKeyFile>ProtoTestConsoleRunner.snk</AssemblyOriginatorKeyFile>
-  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Diagnostics.PerformanceCounter" Version="7.0.0" />
```

Comment:
If this reference is needed, how come it was missing earlier?

Reply:
In .net 6 many types are moved out from base class libraries to nuget packages. I'm unifying both targets to the new API location.